dataverse/webapi/C#-NETx/BulkOperations/UpsertMultiple/README.md

```diff
@@ -0,0 +1,67 @@
+﻿# UpsertMultiple README
+
+This project uses the [UpsertMultiple](https://learn.microsoft.com/power-apps/developer/data-platform/webapi/reference/upsertmultiple) action to demonstrate bulk upsert operations.
+
+## Create table for sample
+
+This sample will create a new table that the code will use. It depends on the common structure for other projects in this solution that is described in [BulkOperations/README.md](../README.md).
+
+When configured for standard tables, this sample demonstrates upserting records using both primary and alternate keys. The `CreateAlternateKey` property in Settings.cs controls whether an alternate key will be created for the table. The default value is `true`. The name of the column defined as an alternate key is `sample_keyattribute`.
+
+Elastic tables have a single alternate key defined and you can't create a new one. When configured for elastic tables, this sample will use the `partitionid` column to uniquely identify records.
```

Comment:
Elastic tables have a predefined alternate key which uses '<tablelogicalname>id' and the 'partitionid'. Both attributes are part of the alternate key.

Reply:
Done